### ISSUE TITLE
Retake development

### ISSUE THREAD

**Issue by Dr4goonD3mon** [WombatFromHell/OriginSteamOverlayLauncher#79]
Hi, I've been using this tool for a while and I love it, my petition is to retake development of the app I use it for practically every non steam game and works amazingly the only app that I don't use with this app are the uwp Xbox games, for these I use glosi but for everything else I use this app and I like to see this app getting support for newer launchers and add new features like suspending the game when the steam overlay is present, this is a great piece of software and need to get support.

**Comment by noggl** [comment]
Are you trying to take ownership of this repo? If you have changes, why not submit a PR instead?

**Comment by WombatFromHell** [comment]
I still use this app a few times a week, and I've not encountered any major issues for awhile with the launchers I use. Unfortunately, I don't really have time to add and test new features, but I'm definitely open to pull-requests and trying to solve issue tickets for breakage on the existing codebase where I can.

**Comment by Dr4goonD3mon** [comment]
> Are you trying to take ownership of this repo? If you have changes, why not submit a PR instead?

No, I just want to this app to get new features because there's a while ago since this app get an update
